src/ucs/type/status.h

```diff
@@ -42,9 +42,23 @@ typedef enum {
     UCS_ERR_TIMED_OUT              = -20,
     UCS_ERR_EXCEEDS_LIMIT          = -21,
     UCS_ERR_UNSUPPORTED            = -22,
+
```

Comment:
You are right. The reason I didn't partition the existing errors is that some errors may end up in more than one group, and making these changes will be part of the implementation - once the API will reach consensus.